sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py

```diff
@@ -350,6 +369,7 @@ def _from_generated(cls, generated):
         props.last_modified = generated.properties.last_modified
         props.etag = generated.properties.etag
         props.quota = generated.properties.quota
+        props.access_tier = generated.properties.access_tier
```

Comment:
In blob we call it blob_tier, do we want to be consistent and call it's share_tier?

Reply:
I actually prefer `access_tier` since this is now a property of a share we shouldn't have a prefix `share`. For example, we call the property `quota` not `share_quota` or `etag` not `share_etag`. I believe for future naming `access_tier` is the better naming convention.

Reply:
it's should be blob_access_tier and share_access_tier or access_tier. I know we have convention to prefix things with blob_/share_ somewhere (not sure if this applies here). But we should also make sure that "access_tier" can be easily matched with other SDKs and REST docs.

Reply:
Looking at https://github.com/Azure/azure-sdk-for-java/pull/15897/files, it appears `access_tier` would be consistent with other SDKs

Reply:
Yeah - I think `access_tier` is a good name. No need for `share` prefix as it's already on `ShareProeprties`.

If we wanted, what we can do in a future Blobs release is to add `access_tier` and stop documenting `blob_tier` - it will always still be there of course so it will never be a breaking change.